demosplan/DemosPlanCoreBundle/Logic/FileResponseGenerator/ZipResponseGenerator.php

```diff
@@ -47,31 +54,66 @@ private function createStreamedResponseForZip(array $file): StreamedResponse
         );
     }
 
-    /**
-     * @throws InvalidDataException
-     * @throws Exception
-     */
     private function fillZipWithData(ZipStream $zipStream, array $file): void
     {
+        $errorMessages = [];
         $temporaryFullFilePath = DemosPlanPath::getTemporaryPath($file['xlsx']['filename']);
         /** @var IWriter $writer */
         $writer = $file['xlsx']['writer'];
-        $writer->save($temporaryFullFilePath);
-        $this->zipExportService->addFileToZipStream(
-            $temporaryFullFilePath,
-            $file['zipFileName'].'/'.$file['xlsx']['filename'],
-            $zipStream
-        );
+        try {
+            $writer->save($temporaryFullFilePath);
+            $this->zipExportService->addFileToZipStream(
+                $temporaryFullFilePath,
+                $file['zipFileName'].'/'.$file['xlsx']['filename'],
+                $zipStream
+            );
+        } catch (FileNotFoundException|FileNotReadableException $e) {
+            $this->logger->critical('Unable to load or read file from path.', [$e]);
```

Comment:
No need to use critical, error or warning should be enough

Reply:
Actually, these errors should never occur and after consulting with Stefan, we decided on critical.

Reply:
Not convinced but I do not insist